src/services/local/index.js

```diff
@@ -111,6 +112,11 @@ export default function(options){
     // Get our initialize service to that we can bind hooks
     const localService = app.service(options.localEndpoint);
 
+    // prevent regular service events from being dispatched
+    if (app.io || app.primus) {
```

Comment:
What were you thinking of for a warning?

```js
if (typeof tokenService.filter === 'function') {
      tokenService.filter(() => false);
    } else if (app.io || app.primus) {
      throw new Error('You cannot override the filter method');
    }
```

How can we detect whether the filter event behaves the way it should other than testing to make sure that events don't get dispatched?